# Record Message

#Tells the datalog node which quad it should record position data for.

bool record_iris1
bool record_iris2
bool record_iris3
bool record_iris4
bool record_iris5
